src/game/camera.c

```diff
@@ -3449,6 +3449,7 @@ void reset_camera(struct Camera *c) {
 
     if (gFirstPersonCamera.enabled) {
         gFirstPersonCamera.yaw = gMarioStates[0].faceAngle[1] + 0x8000;
+        set_mario_action(&gMarioStates[0], ACT_IDLE, 0);
```

Comment:
what are you trying to fix by doing this?

Reply:
I forgot I made this change, but it is supposed to fix Mario getting stuck if the camera resets while in first person